WordPress/src/main/java/org/wordpress/android/ui/mysite/quickstart/QuickStartBlockBuilder.kt

```diff
@@ -1,39 +1,36 @@
 package org.wordpress.android.ui.mysite.quickstart
 
 import org.wordpress.android.R
-import org.wordpress.android.fluxc.store.QuickStartStore
 import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTaskType
 import org.wordpress.android.ui.mysite.MySiteItem.QuickStartBlock
 import org.wordpress.android.ui.mysite.MySiteItem.QuickStartBlock.QuickStartTaskTypeItem
-import org.wordpress.android.ui.prefs.AppPrefsWrapper
+import org.wordpress.android.ui.mysite.QuickStartRepository.QuickStartCategory
 import org.wordpress.android.ui.utils.ListItemInteraction
 import org.wordpress.android.ui.utils.UiString.UiStringRes
 import org.wordpress.android.ui.utils.UiString.UiStringResWithParams
 import org.wordpress.android.ui.utils.UiString.UiStringText
 import javax.inject.Inject
 
-class QuickStartBlockBuilder
-@Inject constructor(
-    private val appPrefsWrapper: AppPrefsWrapper,
-    private val quickStartStore: QuickStartStore
-) {
-    fun build(onItemClick: (QuickStartTaskType) -> Unit): QuickStartBlock {
-        val localSiteId = appPrefsWrapper.getSelectedSite().toLong()
-
+class QuickStartBlockBuilder @Inject constructor() {
+    fun build(
+        categories: List<QuickStartCategory>,
+        onItemClick: (QuickStartTaskType) -> Unit
+    ): QuickStartBlock {
         val taskTypeItems = mutableListOf<QuickStartTaskTypeItem>()
-        taskTypeItems.add(buildQuickStartTaskTypeItem(localSiteId, QuickStartTaskType.CUSTOMIZE, onItemClick))
-        taskTypeItems.add(buildQuickStartTaskTypeItem(localSiteId, QuickStartTaskType.GROW, onItemClick))
+        categories.forEach { category ->
+            taskTypeItems.add(buildQuickStartTaskTypeItem(category, onItemClick))
+        }
```

Comment:
Suggestion (💡): What if the above was replaced with the below:
```
    fun build(
        ...
    ) = QuickStartBlock(
            taskTypeItems = categories.map { buildQuickStartTaskTypeItem(it, onItemClick) }
    )
```

Reply:
Makes it more concise 👍 
Suggestion applied in c05e241